### PR TITLE
Feature/update https setup readme

### PR DESCRIPTION
**Pull request by vskubriev**
### Motivation and context

The documentation section of https install is obsolete and incorrect. Acme.sh client must be used differently. 
This section also requires some clean up.

### How has this been tested?

That has been tested on a new virtual cloud instance once. Hope this prevent new users from wrong way in future. And will point existing users of described manual to right way.

### Checklist

- [X] I submit my changes into the `develop` branch
- [X] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [X] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [X] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by azhavoro** [comment]
@vskubriev Thank you for the contribution!

File: cvat/apps/documentation/installation.md [hunk 1]
```diff
@@ -373,45 +373,42 @@ This depends on the email server you are using and is not covered in this tutori
 [Django SMTP backend configuration](https://docs.djangoproject.com/en/3.1/topics/email/#django.core.mail.backends.smtp.EmailBackend)
 for details.
 
-### Serving over HTTPS
```
Comment (azhavoro) [inline cvat/apps/documentation/installation.md:376]: Please update the table of contents too

**Review by azhavoro — CHANGES_REQUESTED** [review]

**Comment by azhavoro** [comment]
@vskubriev Could you resolve conflicts?

**Comment by vskubriev** [comment]
> @vskubriev Could you resolve conflicts?

Sure. I will do it soon

File: cvat/apps/documentation/installation.md (hunk 1 above)
Comment (vskubriev) [reply cvat/apps/documentation/installation.md:376]: @azhavoro Sorry, I forget to update TOC. Will do it soon.

**Comment by vskubriev** [comment]
@azhavoro hope that's all. please review again

**Review by azhavoro — APPROVED** [review]

**Comment by vskubriev** [comment]
@nmanovic is there all done ?

I can do rebase with current develop branch.

**Comment by nmanovic** [comment]
@azhavoro , could you please review?

**Pull request merged**